manuscript/myst.yml

```diff
@@ -8,40 +8,40 @@ project:
   authors:
     - name: Giuseppe Gallitto
       affiliations:
-        - Center for Translational Neuro- and Behavioral Sciences (C-TNBS), University Medicine Essen, Germany
-        - Department of Neurology, University Medicine Essen, Germany
+        - Center for Translational Neuro- and Behavioral Sciences (C-TNBS), University Hospital Essen, Germany
```

Comment:
University Medicine Essen is the correct one.

Reply:
done